.vsts-dotnet.yml

```diff
@@ -42,11 +42,13 @@ stages:
   displayName: Build
 
   jobs:
-  - template: /eng/common/templates/job/onelocbuild.yml
-    parameters:
-      CreatePr: false
-      LclSource: lclFilesfromPackage
-      LclPackageId: 'LCL-JUNO-PROD-MSBUILD'
+  - {{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
```

Comment:
Don't we almost always want this in our release branches? The loc handback is almost always after we've branched for release.

Reply:
Loc handback occurs continuously now. When you branch for release, you can change this to continue receiving loc handback from those branches.

Reply:
Right, I wasn't considering future release branches. We can modify this to include an `or(XXX, startsWith(refs/heads/vs))`

Reply:
Yeah, would love to filter on the pattern. Don't love the idea of adding a step to branching for release, which is currently git-only.

Reply:
> Don't love the idea of adding a step to branching for release

I also prefer just adding the or instead of having us manually push stuff each release. Will add the condition

Reply:
I don't recommend this as it will cause problems with OneLocBuild, which expects to receive artifacts from only one branch. There's a whole official process for branching for release which you can read [here](https://github.com/dotnet/arcade/blob/main/Documentation/OneLocBuild.md#if-youre-releasing-from-a-branch-other-than-main-including-servicing-branches).

If you want to run two branches at once, you'll need two separate package repositories on the loc side (the `LCL-JUNO` bit). You'll need to file a ticket with the loc team about doing that.

Reply:
Wait, but how would this fix that? You're saying that we need to change the build definition in _both_ the release branch _and_ the main branch for every release?

Reply:
Talked offline. The answer is evidently "yes, this is what we must do".